api/src/feeds/impl/datasets_api_impl.py

```diff
@@ -61,10 +61,26 @@ def apply_bounding_filtering(
             )
         min_latitude, max_latitude = bounding_latitudes_tokens
         min_longitude, max_longitude = bounding_longitudes_tokens
-
+        try:
+            min_latitude = float(min_latitude)
+            max_latitude = float(max_latitude)
+            min_longitude = float(min_longitude)
+            max_longitude = float(max_longitude)
+        except ValueError:
+            raise HTTPException(
+                status_code=400,
```

Comment:
Let's return [422](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422). This is the code that fast API returns when there is a client-side issue with the validity of the request.